functions/update-dynamodb/function.yml

```diff
@@ -0,0 +1,6 @@
+update-dynamodb:
```

Comment:
Forgot to mention in the Issue - could you also add triggering this lambda function by event bridge event every 10 minutes? Instead of hardcoding PK and SK and env variables level, please move them to input of the event bridge event